modules/nodes-cluster-overcommit-project-disable.adoc

```diff
@@ -7,29 +7,60 @@
 [id="nodes-cluster-overcommit-project-disable_{context}"]
 = Disabling overcommitment for a project
 
+ifndef::openshift-rosa,openshift-rosa-hcp,openshift-dedicated[]
 When enabled, overcommitment can be disabled per-project. For example, you can allow infrastructure components to be configured independently of overcommitment.
+endif::openshift-rosa,openshift-rosa-hcp,openshift-dedicated[]
 
-.Procedure
+ifdef::openshift-rosa,openshift-rosa-hcp,openshift-dedicated[]
+If required by your use case, you can disable overcommitment on any project that is not managed by Red Hat. For a list of projects that cannot be modified, see "Red Hat Managed resources" in _Support_.
 
-To disable overcommitment in a project:
+.Prerequisites
+* You are logged in to the cluster using an account with cluster administrator or cluster editor permissions.
 
-ifndef::openshift-rosa,openshift-dedicated[]
-. Create or edit the namespace object file.
-endif::openshift-rosa,openshift-dedicated[]
-// Invalid value: "false": field is immutable, try updating the namespace
-ifdef::openshift-rosa,openshift-dedicated[]
-. Edit the namespace object file.
-endif::openshift-rosa,openshift-dedicated[]
+endif::openshift-rosa,openshift-rosa-hcp,openshift-dedicated[]
 
+.Procedure
+//For OCP and Origin:
+ifndef::openshift-rosa,openshift-rosa-hcp,openshift-dedicated[]
```

Comment:
@laubai Do we need to include `openshift-rosa-hcp` in all `ifdef::openshift-rosa,openshift-dedicated[]` strings? If so, is there a plan to replace them all?

Reply:
The ROSA team is working on this right now as part of the bifurcation ticket: https://issues.redhat.com/browse/OSDOCS-11534

We're reviewing each instance as we go, so no general advisory.